appium/webdriver/extensions/flutter_integration/flutter_commands.py

```diff
@@ -0,0 +1,179 @@
+        Waits for a element to become invisible.
+
+        Args:
+            locator (Union[WebElement, FlutterFinder]): The element to wait for; can be a WebElement or a FlutterFinder.
+            time_out (Optional[float]): Maximum wait time in seconds. Defaults to a predefined timeout if not specified.
+            
+        Returns:
+            None: 
+        """
+        if isinstance(locator, WebElement):
+            opts = {'element': locator, 'timeout': time_out}
+        else:
+            opts = {'locator': locator.to_dict(), 'timeout': time_out}
+        
+        self.execute_flutter_command('waitForAbsent', opts)
+
+    # flutter action commands
+    
+    def perform_double_click(self, element: WebElement, offset: Optional[Tuple[int, int]] = None) -> None:
+        """
+        Performs a double-click on the given element, with an optional offset.
+
+        Args:
+            element (WebElement): The element to double-click on. This parameter is required.
+            offset (Optional[Tuple[int, int]]): The x and y offsets from the element to click at. If not specified, the click is performed at the element's center.
+            
+        Returns:
+            None:
+        """
+        opts = {'origin': element}
+        if offset is not None:
+            opts['offset'] = {'x': offset[0], 'y': offset[1]}
+        self.execute_flutter_command('doubleClick', opts)
+    
+    def perform_long_press(self, element: WebElement, offset: Optional[Tuple[int, int]] = None) -> None:
+        """
+        Performs a long press on the given element, with an optional offset.
+
+        Args:
+            element (WebElement): The element to perform the long press on. This parameter is required.
+            offset (Optional[Tuple[int, int]]): The x and y offsets from the element to perform the long press at. If not specified, the long press is performed at the element's center.
+            
+        Returns:
+            None:
+        """
+        opts = {'origin': element}
+        if offset is not None:
+            opts['offset'] = {'x': offset[0], 'y': offset[1]}
+        self.execute_flutter_command('longPress', opts)
+            
+    def perform_drag_and_drop(self, source: WebElement, target: WebElement) -> None:
+        """
+        Performs a drag-and-drop operation from a source element to a target element.
+
+        Args:
+            source (WebElement): The element to drag from.
+            target (WebElement): The element to drop onto.
+            
+        Returns:
+            None:
+        """
+        self.execute_flutter_command('dragAndDrop', {'source': source, 'target': target})
+        
+    def scroll_till_visible(self, scroll_to: FlutterFinder, scroll_direction: Optional[ScrollDirection] = ScrollDirection.DOWN, **opts: Any) -> WebElement:
+        """
+        Scrolls until the specified element becomes visible.
+
+        Args:
+            scroll_to (FlutterFinder): The Flutter element to scroll to.
```

Comment:
there is no need to repeat types in doctrings

Reply:
are you suggesting me to remove `(FlutterFinder)`?

Reply:
yes, do it for all args where the time is set explicitly upon declaration

Reply:
Any reason ? I see it is useful when I see documentation 
<img width="518" alt="Screenshot 2024-09-21 at 6 55 05 PM" src="https://github.com/user-attachments/assets/5bfd4ed7-ad61-4e6c-8e4b-e55e39c68038">